src/components/Dialog/DialogContentInner.tsx

```diff
@@ -66,7 +86,9 @@ export const DialogContentInner: FC<Props> = ({
       >
         <Wrapper ref={domRef} themes={theme}>
           <Background onClick={onClickOverlay} themes={theme} />
-          <Inner themes={theme} {...props}>
+          <Inner ref={innerRef} themes={theme} role="dialog" aria-modal="true" {...props}>
+            {/* dummy element for focus management. */}
+            <div ref={focusTarget} tabIndex={-1} aria-label={ariaLabel}></div>
```

Comment:
フォーカスマネジメントのためのダミー要素です。ダイアログを開いた時に、最初のタブ移動可能な要素にフォーカスをあてる方法だとダイアログのコンテンツが長い場合にスクロールされてしまうためコンテンツの冒頭にフォーカスを持ってくる方法をとりました。

I don’t choose the way to focus first tabbable element in contents. because when dialog contents is long, scrolled dialog. So the way to focus top of contents is better.